src/content/booknotes/lying.mdx

```diff
@@ -13,88 +13,88 @@ amazonLink: ""
+Hence lying is bad and we shouldn't do it, in any circumstances where we expect to have any sort of ongoing relationship with the people lied to. Only exceptions are extreme, the lives of spies, or war times, or when murderers ask you for help etc.
 
-## Detailed Notes 
+## Detailed Notes
 
 ## Chapter 1 - What is a Lie?
 
-> Not all acts of deceptions are lies. 
+> Not all acts of deceptions are lies.
 
-Lying is not telling the truths if other expect you to tell the truth. It's about the intent to deceive. 
+Lying is not telling the truths if other expect you to tell the truth. It's about the intent to deceive.
 
-When lying we damage the person being lied to. Nobody wants to be on the receiving role of a lie. 
+When lying we damage the person being lied to. Nobody wants to be on the receiving role of a lie.
 
 ## Chapter 2 - The Mirror of Honesty
 
-> Honesty is a gift we can give to others. 
+> Honesty is a gift we can give to others.
 
-Being honest ist simple. It's being yourself. Its being. 
+Being honest ist simple. It's being yourself. Its being.
 
-Because lies can hide the truth, they can shield us from uncomfortable realities. Addictions, Broken Relationships or even Things we are too afraid to do, but too proud to admit. This way lying forces us to grow. 
+Because lies can hide the truth, they can shield us from uncomfortable realities. Addictions, Broken Relationships, or even Things we are too afraid to do, but too proud to admit. This way, lying forces us to grow.
 
 ## Chapter 3 - Two Types of Lies
 
-Actions that take energy to do, are judged more harshly if morally reprehensible. It's different to steal, than to not pay back money wrongly received. 
+Actions that take energy to do are judged more harshly if morally reprehensible. It's different to steal than to not pay back money wrongly received.
 
-Lies fall into the same category. Omission and Commission. 
+Lies fall into the same category. Omission and Commission.
 
-Lying is denying access to reality. And that is something harmful. 
+Lying is denying access to reality. And that is harmful.
 
-False encouragement is stealing away time that could have been used better. 
+False encouragement is stealing away time that could have been used better.
 
-Lies in the medical context forsake the chance of facing the horrible truth of death, and sharing it with the family, opens up the world a little more, for wisdom, compassion and love to be shared, although it creates pain. 
+Lies in the medical context forsake the chance of facing the horrible truth of death. Sharing it with the family opens up the world a little more for wisdom, compassion and love to be shared, although it creates pain.
 
 ## Chapter 4 - Trust
 
-People stop trusting people, if they observed them lying in the past. 
+People stop trusting people if they observed them lying in the past.
 
-When found out, lies produce huge embarrassment, that is never worth the benefit of the lie. 
```

Comment:
A lot of the commas in this booknote make the sentences harder to read. Not all pauses in speech require a comma.

Regarding the comma before `that`s: https://ell.stackexchange.com/questions/9154/use-of-commas-before-that

> As a rule of thumb, you don't use a comma before a clause that begins with that.